lnrpc/lightning.proto

```diff
@@ -1111,9 +1111,8 @@ message EstimateFeeResponse {
     // The total fee in satoshis.
     int64 fee_sat = 1;
 
-    // Deprecated, use sat_per_vbyte.
-    // The fee rate in satoshi/vbyte.
-    int64 feerate_sat_per_byte = 2 [deprecated = true];
+    // The fee rate in satoshi/kweight.
+    int64 sat_per_kweight = 2;
```

Comment:
I propose to use another ID for this field and not to change the deprecated field, not to break binary backwards compatibility. If someone has an old client and runs against new server (or vice-versa), they can send a request with this field, which has changed meaning, resulting in huge overpayment or underpayment of fee.